doc/api/esm.md

```diff
@@ -456,16 +456,24 @@ Conditional exports can also be extended to exports subpaths, for example:
   "exports": {
     ".": "./main.js",
     "./feature": {
-      "browser": "./feature-browser.js",
+      "node": "./feature-node.js",
       "default": "./feature.js"
     }
   }
 }
 ```
 
 Defines a package where `require('pkg/feature')` and `import 'pkg/feature'`
-could provide different implementations between the browser and Node.js,
-given third-party tool support for a `"browser"` condition.
+could provide different implementations between Node.js and other JS
+environments.
+
+When using environment branches, include
+a `"defaut"` condition where possible. Always providing a `"default"` condition
+ensures that any unknown JS environments are able to use a universal
+implementation where possible, which helps avoid JS environments from having to
+pretend to be existing environments to support packages with conditional
+exports. Where possible, `"node"` and `"default"` conditions can be preferable
+to `"browser"` conditions.
```

Comment:
nit: adjusting line wrapping

```suggestion
When using environment branches, include a `"defaut"` condition where
possible. Always providing a `"default"` condition ensures that any
unknown JS environments are able to use a universal implementation where
possible, which helps avoid JS environments from having to pretend to be
existing environments to support packages with conditional exports. Where
possible, `"node"` and `"default"` conditions can be preferable to `"browser"`
conditions.
```

Reply:
s/defaut/default

Reply:
Thanks, I've actually gone through and done a little rewording here as well.